runtime/common/Cargo.toml

```diff
@@ -28,6 +29,7 @@ pallet-transaction-payment-rpc-runtime-api = { branch = "polkadot-v0.9.32", defa
 pallet-treasury = { branch = "polkadot-v0.9.32", default-features = false, git = "https://github.com/paritytech/substrate" }
 pallet-utility = { branch = "polkadot-v0.9.32", default-features = false, git = "https://github.com/paritytech/substrate" }
 pallet-vesting = { branch = "polkadot-v0.9.32", default-features = false, git = "https://github.com/paritytech/substrate" }
+xcm = { branch = "release-v0.9.32", default-features = false, git = "https://github.com/paritytech/polkadot", optional = true }
```

Comment:
What do we need that for? I can't see any newly added code using that.

Reply:
Eagle eye! I once used that, but then did throw away the code that used it.

462e24a1f3a5ff1890d1900eb91154ab702dbf80